app/representers/api/v1/user_representer.rb

```diff
@@ -45,6 +45,12 @@ class UserRepresenter < Roar::Decorator
              readable: true,
              writeable: false
 
+    property :is_test?,
+             as: :is_test,
+             type: :boolean,
+             readable: true,
+             writeable: false
```

Comment:
Can you add a `schema_info` description that says: "If true, the user is not a real OpenStax end user but instead an internal test user."